impress-org/givewp
# Fix: exclude Tests folder from strauss

Resolves [GIVE-1136]

## Description



While using Strauss for the `"symfony/http-foundation"`, it was copying over its Tests folder.  This adds the Tests folder to the list of files to exclude during copy.

## Affects



- The composer build process
- This should not have an affect on production as we are only removing a dev testing folder

## Visuals



<img width="1431" alt="Screenshot 2024-08-20 at 9 42 17 AM" src="https://github.com/user-attachments/assets/a75ca188-6abb-43e9-9b89-73d8f417d33b">

## Testing Instructions



Build: https://github.com/impress-org/givewp/actions/runs/10472792166

Run composer install (delete vendor-prefixed if already exists) and make sure this `Tests` folder is not present:

<img width="436" alt="Screenshot 2024-08-20 at 9 06 08 AM" src="https://github.com/user-attachments/assets/f992d2ac-88f5-4536-a437-a94b27a22847">

## Pre-review Checklist






[GIVE-1136]: https://stellarwp.atlassian.net/browse/GIVE-1136?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
